DataScience/Analytics/beta/check_performance.py

```diff
@@ -181,7 +194,9 @@ def _check_performance(data, verbose=True):
         print('Correct Match not in the Set:', len(incorrect))
 
     results.append(len(correct_in_set))
+    results.append(new_uprns)
```

Comment:
To the same array you append first an int (`len`) and then another array, is that correct ?